test_schemas/en/test_routing_to_section_end.json

```diff
@@ -23,21 +23,29 @@
             "type": "string"
         }
     ],
+    "questionnaire_flow": {
+        "type": "Linear",
+        "options": {
+            "summary": {
+                "collapsible": false
+            }
+        }
+    },
     "sections": [
         {
-            "id": "test-skipping-section",
+            "id": "test-section",
```

Comment:
Probably doesn't need test prefix for all ids

Reply:
I guess but don't really want to spend more time on ids. I would say we need a style guide and/or a linter. This is an issue with a **lot** of our schemas and I think it makes sense to address it all together and have a style guide.